plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -99,16 +101,21 @@ servers = [
 # include_query = []
 
 ## A list of queries to explicitly ignore.
-# exclude_query = []
+## Query Store queries (AzureSQLDBQueryStoreRuntimeStatistics and AzureSQLDBQueryStoreWaitStatistics) are excluded by default.
+## If you enable Query Store queries, the collection interval should comply with the following restriction: 15m =< collection interval <= 2h. 
+## Intervals shorter than 15m may cause higher performance impact on source. Intervals longer than 2h may cause some Query Store data not to be collected.
+# exclude_query = ["AzureSQLDBQueryStoreRuntimeStatistics", "AzureSQLDBQueryStoreWaitStatistics"]
 
 ## Queries enabled by default for database_type = "AzureSQLManagedInstance" are - 
 ## AzureSQLMIResourceStats, AzureSQLMIResourceGovernance, AzureSQLMIDatabaseIO, AzureSQLMIServerProperties, AzureSQLMIOsWaitstats, 
 ## AzureSQLMIMemoryClerks, AzureSQLMIPerformanceCounters, AzureSQLMIRequests, AzureSQLMISchedulers
 
 # database_type = "AzureSQLManagedInstance"
 
+## A list of queries to include.
 # include_query = []
 
+## A list of queries to explicitly ignore.
 # exclude_query = []
```

Comment:
See my comment in `README.md`.

Reply:
@srebhan, please look at my reply on your comment under README.md.